### PR TITLE
Alerts webhooks are introduced

### PR DESCRIPTION
**Pull request by ekremsekerci**
Initial planned release, 18 August

As a draft because:

* Need to check prerequisites and update the corresponding section in the document
* Release note should be updated when the release date is decided

Still needs a proofread:

* In Mendix webhooks are used for multiple purposes. I've just updated `receive-alerts.md` since it's the page directly for alerts for mendix apps. However, I'm not sure if some other page needs to be added or updated.
* My wording can be reviewed. Not sure if I explained everything clearly and in a good way.

### CODE REVIEW

File: content/en/docs/developerportal/operate/monitoring-application-health/receive-alerts.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ tags: ["Technical Contact","Settings","Alerts","Developer Portal"]
 
 ## 1 Introduction
 
+Alerts for Mendix applications can be received in two ways; via email or using webhooks. You can find detailed information for both of the options below.
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/operate/monitoring-application-health/receive-alerts.md]: Is this only for apps running on the Mendix Cloud? If so, I wonder if we should make it clearer that this is only for apps running on the Mendix Cloud?

**Review by MarkvanMents — COMMENTED** [review]
Thanks @ekremsekerci 
Can you let me know when this is ready to be reviewed. Thanks.

File: content/en/docs/developerportal/operate/monitoring-application-health/receive-alerts.md (hunk 1 above)
Comment (ekremsekerci) [reply content/en/docs/developerportal/operate/monitoring-application-health/receive-alerts.md]: Let me check with the team

**Comment by ekremsekerci** [comment]
> Thanks @ekremsekerci Can you let me know when this is ready to be reviewed. Thanks.

Hey @MarkvanMents, the implementation is also ready. It's being reviewed and peer tested nowadays. We are planning to release it this week. So, I think this can be reviewed/proofread as well. I only need to update the release date and prerequisites which I'm not quite sure of yet.

**Comment by MarkvanMents** [comment]
@ekremsekerci 
I've proofread this. Looks good to go once the To Do items are resolved.
I also discovered that the descriptions for the email alerts was out-of-date, so I've taken the opportunity to update that at the same time.

File: content/en/docs/developerportal/operate/monitoring-application-health/receive-alerts.md (hunk 1 above)
Comment (ekremsekerci) [reply content/en/docs/developerportal/operate/monitoring-application-health/receive-alerts.md]: Hi Mark, yes for the current implementation we are doing now, it's only for Mendix Cloud

**Comment by ekremsekerci** [comment]
Hi @MarkvanMents, thanks for the proofread. For the todo items:

1. I removed the prerequisites section since I realized there is already a generic page explaining webhooks system and this information should go there if there are prerequisites. And also it'd be good to add this page as a generic webhooks reference somewhere in the `receive-alerts` page. I was not sure where should we add, can you help me with this? https://docs.mendix.com/developerportal/deploy/webhooks/

2. Since everybody is busy in the team this week we coudn't peer test this and cannot release this week. I'll let you know once it's released so that we can update the date here and publish the docs

**Comment by ekremsekerci** [comment]
Hey @MarkvanMents , just added some information on generic webhooks page and cross referenced both pages. You can take a look when you have some time. Will let you know once it's released so that we can release the docs too.

**Review by MarkvanMents — APPROVED** [review]
@ekremsekerci 
Looks good - just needs the release date updated when it is released.
Please let me know when it is released and I'll update the date and publish.
Thanks.

**Comment by ekremsekerci** [comment]
Hey @MarkvanMents, we released this yesterday and I updated the dates and merged from upstream. Could you please take a look at it one last time?

**Comment by MarkvanMents** [comment]
@ekremsekerci 
I've just changed the date so that we are consistent about it being the 4th.
OK to publish?

**Comment by ekremsekerci** [comment]
Hi @MarkvanMents, yes, ok to publish. Thank you

**Pull request merged**